.github/workflows/build_n_deploy_dev.yaml

```diff
@@ -33,33 +30,8 @@ jobs:
           NODE_AUTH_TOKEN: ${{ secrets.READER_TOKEN }}
         run: yarn test
 
-  bygg-yarn-for-dependabot:
```

Comment:
Gitt at vi kun kjører denne ved workflow_dispatch vil det ikke skje at actor er dependabot 🤔

Reply:
Og når vi kjører den med workflow_dispatch er det nettopp for å bygge ut til miljø, derfor kjører vi `bygg` og `deploy`-jobbene